main.py

```diff
@@ -1,13 +1,18 @@
 import comfy.options
 comfy.options.enable_args_parsing()
 
+# Intercept logs for error reporting
+import sys
+from app.log_interceptor import LogInterceptor
+sys.stdout = LogInterceptor(sys.stdout)
```

Comment:
All the logging in core comfy uses the logging python module so is this really the best way of doing this?

Reply:
> All the logging in core comfy uses the logging python module so is this really the best way of doing this?

We also want to intercept all stdout from custom nodes which we don't have control over. Custom nodes can just call print for console message. Currently ComfyUI-Manager is doing the same stdout object hijack to save output log file.